go/api/commands.go

```diff
@@ -0,0 +1,19 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+// GenericCommands defines an interface to handle generic command and transaction requests for a standalone Redis client.
+type GenericCommands interface {
```

Comment:
Right now it isn't used, so it might be unnecessary. Unlike Java, you can't explicitly declare that RedisClient implements GenericCommands. However, if a function had a GenericCommands parameter, you would be able to pass it a RedisClient. We don't have any such functions yet, and I suppose this would only be useful if we had multiple types that implement GenericCommands, which won't be the case so far. Should I just remove it?